src/components/DetailsDisplay/DetailsDisplay.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./DetailsDisplay.css";
```

Comment:
I would usually put library imports first e.g. `import A from 'library'`, then the local ones e.g. `import A from './A'`